vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogTestPageIT.java

```diff
@@ -205,6 +204,26 @@ public void openEmptyDialog_dialogContentHasWidth() {
                 greaterThan(endpoint));
     }
 
+    @Test
+    public void verifyDialogFullSize() {
+        findElement(By.id("div-in-dialog-button")).click();
+        WebElement overlay = getInShadowRoot(getOverlayContent(),
+                By.id("overlay"));
+        Assert.assertTrue(
+                overlay.getAttribute("style").contains("width: 100%;"));
+        Assert.assertTrue(
+                overlay.getAttribute("style").contains("height: 100%;"));
+
+        WebElement content = overlay.findElement(By.id("content"));
+
+        Assert.assertTrue(
+                content.getAttribute("style").contains("width: 100%;"));
```

Comment:
The fact that the element has style `height: 100%` doesn't prove it's actually taking full size. You'll need to get the effective size of the element.

Reply:
Done.